### PR TITLE
docs(js): remove vdm with generator for tutorial

### PR DESCRIPTION
**Pull request by ZhongpinWang**
## What Has Changed?

Closes SAP/cloud-sdk-backlog#817.

Remove the VDM package used in the tutorial with the generator.

## Manual Checks?

- [x] Check spelling and grammar, e.g., using Grammarly.
- [x] Verify links still work, e.g., if `id` or the name of a file is changed.
- [x] Update the [feature matrix](https://github.com/SAP/cloud-sdk/blob/main/docs/components/data/features.js), e.g., if a new feature is added.

### CODE REVIEW

File: docs-js/tutorials/getting-started/2-execute-odata-request.mdx [hunk 1]
```diff
@@ -89,19 +89,41 @@ npm run start:dev
...
+       "npmPackageName": "business-partner-service"
+     }
+   }
+   ```
+6. Generate the `BusinessPartner` service.
+   ```bash
+   npx generate-odata-client --inputDir service-specifications --outputDir services
+   ```
+
+The generated client library can be found in `services/business-partner-service`.
```
Comment (FrankEssenberger) [inline docs-js/tutorials/getting-started/2-execute-odata-request.mdx:122]: [pp] use active form: The generated library is in the folder.... OR you find the client library in ....

File: docs-js/tutorials/getting-started/2-execute-odata-request.mdx [hunk 2]
```diff
@@ -89,19 +89,41 @@ npm run start:dev
...
 
 :::note
-The SAP Cloud SDK for JavaScript offers packages for each OData service exposed by SAP S/4HANA Cloud.
-You can find a list of these services in the [SAP API Business Hub](https://api.sap.com/).
+A list of services exposed by SAP S/4HANA Cloud can be found in the [SAP API Business Hub](https://api.sap.com/).
```
Comment (FrankEssenberger) [inline docs-js/tutorials/getting-started/2-execute-odata-request.mdx:125]: [pp] also here active form

**Review by FrankEssenberger — APPROVED** [review]
LGTM please adjust the tense from passive to active

**Review by deekshas8 — APPROVED** [review]
LGTM

**Pull request merged**